client/src/components/ReviewForm.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom"
+
+import translateServerErrors from "../services/translateServerErrors.js"
+
+const ReviewForm = (props) => {
+
+    const [newReview, setNewReview] = useState({
+        textBody: "",
+        rating: ""
+    })
+
+    const [errors, setErrors] = useState({})
+    const site = props.site
+
+    const addNewReview = async (formData) => {
+        try {
+            const response = await fetch(`/api/v1/sites/${site.id}`, {
```

Comment:
this path is wrong, as it suggests an endpoint for creating sites, not reviews. 
It should be `/api/v1/sites/${site.id}/reviews`